internal/configuration/curves.go

```diff
@@ -15,6 +15,7 @@ type LinearCurveConfig struct {
 
 const (
 	FunctionAverage = "average"
+	FunctionDelta   = "delta"
```

Comment:
We have to add this new possibile function type to the README and example config file documentation too.